### PR TITLE
[react] ConnectWallet v3 UI 

### PR DESCRIPTION
**Pull request by MananTank**

### CODE REVIEW

File: .changeset/brown-elephants-boil.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/wallets": patch
+---
+
+ConnectWallet v3 updates
```
Comment (iketw) [inline .changeset/brown-elephants-boil.md:5]: Can we remove this file and keep the other one?

File: .changeset/polite-squids-poke.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"@thirdweb-dev/react": minor
+"@thirdweb-dev/react-core": patch
+"@thirdweb-dev/react-native": patch
+---
+
+ConnectWallet v3 updates
```
Comment (iketw) [inline .changeset/polite-squids-poke.md:7]: Can update this changelog with the api changes and how they can be used? Like the new theme, new props, etc

File: .changeset/brown-elephants-boil.md (hunk 1 above)
Comment (MananTank) [reply .changeset/brown-elephants-boil.md:5]: sure

File: .changeset/polite-squids-poke.md (hunk 2 above)
Comment (MananTank) [reply .changeset/polite-squids-poke.md:7]: sure

File: packages/react-core/src/core/types/wallet.ts [hunk 3]
```diff
@@ -127,6 +133,11 @@ export type ConnectUIProps<I extends WalletInstance = WalletInstance> = {
    * List of all supported wallets including your wallet.
    */
   supportedWallets: WalletConfig[];
+
+  /**
+   * Size of the modal
+   */
+  modalSize: "compact" | "wide";
```
[the remark this answers is not in the document]
Comment (MananTank) [reply packages/react-core/src/core/types/wallet.ts:140]: Ok, Let me define a separate type for web instead of modifying the type in core

**Pull request merged**